component/storage/mongodb/mongodb_test.go

```diff
@@ -69,21 +94,7 @@ func waitForMongoDBToStart() error {
 
 	db := mongoClient.Database("test")
 
-	timeout := time.After(10 * time.Second)
-
-	for {
-		select {
-		case <-timeout:
-			return fmt.Errorf("timeout: couldn't reach mongodb server")
-		default:
-			err := db.Client().Ping(context.Background(), nil)
-			if err != nil {
-				return err
-			}
-
-			return nil
-		}
-	}
+	return db.Client().Ping(context.Background(), nil)
```

Comment:
should you use a context with timeout here?

Reply:
Sure, fixed.